tanner/redis_client.py

```diff
@@ -15,6 +15,7 @@ class RedisClient:
         try:
             host = TannerConfig.get('REDIS', 'host')
             port = TannerConfig.get('REDIS', 'port')
+            expiry_time = TannerConfig.get('REDIS', 'expiry_time')
```

Comment:
For now this variable is not used at all. If you want to add this variable to config, please add the functionality as well.

Reply:
It will be used to auto-delete sessions in redis server so as to make free space

Reply:
I understand the purpose of this variable :) Do you want to implement this functionality? If yes, please make it as one PR.

Reply:
Yes I want to implement this functionality. Okay, I will make it as another PR. Is there any better way to implement this functionality ?Suggestions pls